java/src/jmri/jmrix/loconet/LnOpsModeProgrammer.java

```diff
@@ -227,12 +227,10 @@ public void writeCV(String CV, int val, ProgListener pL) throws ProgrammerExcept
             log.debug("  Message {}", m);
             memo.getLnTrafficController().sendLocoNetMessage(m);
             lncvAccessTimer.start();
-        } else if (getMode().equals(LnProgrammerManager.LOCONETOPSBOARD)) {
```

Comment:
It does nothing of the sort.  It combines that if clause with the else directly after it, because they both do the same thing now. See the comment there.